test/e2e/network/util.go

```diff
@@ -58,13 +59,18 @@ func VerifySCTPModuleLoadedOnNodes(f *framework.Framework, nodes *v1.NodeList) b
 	cmd := "lsmod | grep sctp"
 	for _, node := range nodes.Items {
 		framework.Logf("Executing cmd %q on node %v", cmd, node.Name)
-		err := hostExec.IssueCommand(cmd, &node)
+		result, err := hostExec.IssueCommandWithResult(cmd, &node)
 		if err != nil {
 			framework.Logf("sctp module is not loaded or error occured while executing command %s on node: %v", cmd, err)
 			return false
 		} else {
-			framework.Logf("the sctp module is loaded on node: %v", node.Name)
-			return true
+			for _, line := range strings.Split(result, "\n") {
+				if !strings.Contains(line, "xt_sctp") && !!strings.Contains(line, "nf_conntrack_proto_sctp") {
```

Comment:
sorry if this is a dumb question but if I do execute in my host I have
```
linux-6my5:~ # lsmod | grep sctp
sctp_diag              16384  0
sctp                  352256  39 sctp_diag
inet_diag              20480  4 raw_diag,tcp_diag,sctp_diag,udp_diag
libcrc32c              16384  5 ip_vs,nf_conntrack,xfs,nf_nat,sctp
```

is this `if` looking for a line that does not contain `xt_sctp` and `nf_conntrack_proto_sctp`?

Reply:
Well, that `if` tries to filter out those modules that are not harmful from userspace SCTP stack perspective. But the best would be to check if the `sctp` module is loaded or not.

Reply:
Coded a more sophisticated one with regex.